packages/lesswrong/components/posts/PostsItem2.tsx

```diff
@@ -436,7 +437,7 @@ const PostsItem2 = ({
           >
                 {tagRel && <Components.PostsItemTagRelevance tagRel={tagRel} post={post} />}
                 <PostsItem2MetaInfo className={classes.karma}>
-                  <PostsItemKarma post={post} />
+                  {post.isEvent ? <AddToCalendarIcon post={post} /> : <PostsItemKarma post={post} />}
```

Comment:
Interesting, so event-posts have an add-to-calendar icon replacing their karma score. Seems like a fine change, but I do want to flag that we're making it.

Reply:
See discussion here: https://github.com/centre-for-effective-altruism/EAForum/pull/214